src/content/en/2021/caching.md

```diff
@@ -1,20 +1,489 @@
+
+Based on the **Lighthouse caching TTL** score, we have seen an improvement in pages ranked with a perfect score of 100 increase from 3.3% in 2020 to 4.4% in 2021.
+
+The score reflects whether the pages can benefit from additional caching policy improvements. Even though we are excited to see 31% of pages scoring above the 50 percentile score, a large potential for improvement exists for the 52% of pages that are ranking below the 25 percentile.
+
+{{ figure_markup(
+  image="lighthouse-caching-ttl-scores.png",
+  caption="Distribution of Lighthouse caching TTL scores.",
+  description="A bar chart showing the distribution of Lighthouse audit scores for the `uses-long-cache-ttl` for mobile web pages. 36.2% of the responses have a score less than 0.10, 25.6% have a score between 0.10-0.39, 20.3% have a score between 0.40-0.79, and 12.1% have a score between 0.80-0.99. 4.4% have a score of 1 and 1.3% have no score.",
+  chart_url="https://docs.google.com/spreadsheets/d/e/2PACX-1vSGgVDZ9RkFQLmk5C3_siIcH-8macUEZMobcC0o1z8frYj8NOkI_C2s_yE5ppMdxDAD5INjNsCBa3h1/pubchart?oid=1825783189&format=interactive",
+  sheets_gid="2098771743",
+  sql_file="cache_ttl_lighthouse_score.sql"
+) }}
+
+This makes us consider that even though web pages have some level of caching, the way the policies are used is outdated and not optimized to the latest state of their products.
+
+Based on **Lighthouse wasted bytes** from 2020 to 2021, there was a 3.28% improvement in wasted bytes across all audited pages on repeated views. This lowers the percentage of pages that waste 1 MB from 42.8% to 39.5%, showing a considerable trend from the community towards building products that are less costly for international users with paid internet data plans.
+
+{{ figure_markup(
+  image="lighthouse-caching-byte-savings.png",
+  caption="Distribution of potential byte savings from caching.",
+  description="A bar chart showing the distribution of potential byte savings from the Lighthouse caching audit for mobile web pages. 59.1% of the responses have a size saving less than 1 MB, 19.8% have a saving between 1-2 MB, 7.1% have a saving between 2-3 MB, and 3.9% have a saving between 3-4 MB. 8.7% have a saving of 4 MB or more.",
+  chart_url="https://docs.google.com/spreadsheets/d/e/2PACX-1vSGgVDZ9RkFQLmk5C3_siIcH-8macUEZMobcC0o1z8frYj8NOkI_C2s_yE5ppMdxDAD5INjNsCBa3h1/pubchart?oid=1045307868&format=interactive",
+  sheets_gid="469776025",
+  sql_file="cache_wastedbytes_lighthouse.sql"
+) }}
+
+The current percentage of pages audited that have 0 wasted bytes is still relatively low at 1.34%. In the coming years, we're looking forward to seeing an increase in that percentage as the community continues to focus on optimizing web performance.
+
+## Conclusion
+
+You may recognize this quote by Phil Karlton, "*There are only two hard things in Computer Science: cache invalidation and naming things.*" And in all honesty I have always wondered why caching is so hard.
```

Comment:
Edit:

Phil Karlton famously said, "*There are only two hard things in Computer Science: cache invalidation and naming things.*" And in all honesty I have always wondered why caching is so hard.

Reply:
Can we link Phil's name to either https://www.karlton.org/2017/12/naming-things-hard/ or to https://www.karlton.org/karlton/?

Also, I would change to "The late, great Phil Karlton famously said..." But that's just me :)

Reply:
Great catch!

Reply:
Edit:

The late, great [Phil Karlton](https://www.karlton.org/karlton/) famously said, "_There are only two hard things in Computer Science: cache invalidation and naming things._" And in all honesty I have always wondered why caching is so hard.